pkgs/swiftgen/swift2objc/lib/src/transformer/transformers/transform_method.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:swift2objc/src/ast/_core/shared/parameter.dart';
+import 'package:swift2objc/src/ast/_core/shared/referred_type.dart';
+import 'package:swift2objc/src/ast/declarations/compounds/class_declaration.dart';
+import 'package:swift2objc/src/transformer/transform.dart';
+import 'package:swift2objc/src/transformer/transformers/transform_referred_type.dart';
+
+ClassMethodDeclaration tranformMethod(
+  ClassMethodDeclaration originalMethod,
+  TransformationMap transformationMap,
+) {
+  final transformedParams = originalMethod.params
+      .map((param) => _tranformParamter(param, transformationMap))
+      .toList();
+
+  final transformedMethod = ClassMethodDeclaration(
+    id: originalMethod.id,
+    name: originalMethod.name,
+    returnType: transformReferredType(
+      originalMethod.returnType,
+      transformationMap,
+    ),
+    params: transformedParams,
+    hasObjCAnnotation: true,
+  );
+
+  transformedMethod.statements = <String>[
+    _generateMethodCall(originalMethod, transformedMethod)
+  ];
+
+  transformationMap[originalMethod] = transformedMethod;
```

Comment:
Do methods need to be in the `transformationMap`?

Reply:
um well no not really. I removed it.